tests/common/devices.py

```diff
@@ -318,9 +318,9 @@ def get_image_info(self):
                   return a dictionary of "current, next, installed_list"
         """
         output = self.command("sonic_installer list")["stdout"]
-        lines  = output.split('\n')
+        lines  = output.splitlines()
```

Comment:
not needed any more

Reply:
Why is this not needed? I believe this was the change that Danny requested below.

Reply:
oh, if we take Tamer's suggestion to use stdout_lines

Reply:
My bad. I thought the var 'lines' wasn't used anymore. I was looking at Danny's suggested changes 'for line in output.splitlines()'